run.js

```diff
@@ -6,14 +6,29 @@ const co = require('co');
 exports.disconnect = disconnect;
 
 let sailor;
+let disconnectRequired;
 
 co(function* putOutToSea() {
     sailor = new Sailor(settings);
+
+
+    if (!!settings.HOOK_SHUTDOWN) {
+        disconnectRequired = false;
+        sailor.reportError = () => {
```

Comment:
`reportError` sends some data into `amqp`.
In `shutdown` mode sailor doesn't listen to `amqp`.
So this workaround is in order to disable error reporting.